data_structures/Cargo.toml

```diff
@@ -10,7 +10,7 @@ workspace = ".."
 bls-signatures-rs = "0.1.0"
 bech32 = "0.7.2"
 byteorder = "1.3.4"
-cbor-codec = "0.7.1"
+cbor-codec = { git = "https://gitlab.com/adansdpc/cbor-codec.git", branch = "feat/ldexpf-shim" }
```

Comment:
Can we fork this as `witnet/cbor-codec`? I think github has a feature to import foreign repos.

Reply:
I forked it on GitLab with a view to doing a pull request there. Do you think I should still replicate the repo on GitHub for the sake of having all our stuff in the same place?

Reply:
Yes that's what I do when forking. Ideally `cbor-codec` would be updated to `0.7.2` with that fix, but I think that project is unmaintained.

Reply:
Done!